l10n_ch_hr_payroll/__manifest__.py

```diff
@@ -4,6 +4,7 @@
 #  Module: l10n_ch_hr_payroll
 #
 #  Created by sge@open-net.ch
+#  Updated by lfr@open-net.ch (2017)
```

Comment:
Header for this file https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3